test/test_files/exceptions/binder/binder_error.test

```diff
@@ -5,6 +5,16 @@
 
 -CASE BinderError
 
+-LOG EmptyCreateClause
+-STATEMENT MATCH (a:person) CREATE (a);
+---- error
+Binder exception: Cannot resolve any node or relationship to create.
```

Comment:
I wonder if we can throw error message same as `Binder exception: Node table PERSON does not exist`. which seems more informative.